tracing-error/README.md

```diff
@@ -45,16 +46,20 @@ The crate provides the following:
 
 *Compiler support: requires `rustc` 1.39+*
 
-## Usage
+## Feature Flags
+
+- `traced-error` - Enables the [`TracedError`] type and related Traits
+    - [`InstrumentResult`] and [`InstrumentError`] extension traits, which
+    provide an [`in_current_span()`] method for bundling errors with a
+    [`SpanTrace`].
+    - [`ExtractSpanTrace`] extension trait, for extracting `SpanTrace`s from
+    behind `dyn Error` trait objects.
```

Comment:
I would probably put this below "Usage"?

Reply:
K, moved, should I do the same thing in lib.rs?

Reply:
IMO, yes.